rpc/backend/backend.go

```diff
@@ -189,4 +199,6 @@ func NewBackend(
 		allowUnprotectedTxs: allowUnprotectedTxs,
 		indexer:             indexer,
 	}
+	b.processBlocker = b.processBlock
```

Comment:
is this useless?

Reply:
need for test

Reply:
I mean the assignment, `b.processBlock` is always `nil` here

Reply:
why? [b.processBlock](https://github.com/mmsqe/ethermint/blob/c0312a4afe2959c0a01d22f2848816e0aeb504a1/rpc/backend/utils.go#L162) is a function

Reply:
```
b := &Backend{
```

`b` is just constructed, where the `processBlock` field is not set, right?

Reply:
but processBlocker is property, b.processBlock =/= b.processBlocker

Reply:
oh, `processBlock` is a method.